src/window/error_window.rs

```diff
@@ -0,0 +1,488 @@
+use skia_safe::{
+    canvas::{Canvas, SaveLayerRec},
+    colors::{BLACK, WHITE},
+    textlayout::{
+        FontCollection, Paragraph, ParagraphBuilder, ParagraphStyle, TextHeightBehavior, TextIndex,
+        TextStyle,
+    },
+    Color4f, FontMgr, Paint, Point, Rect, Size,
+};
+use strum::IntoEnumIterator;
+use strum::{EnumCount, EnumIter};
+use winit::{
+    dpi::PhysicalSize,
+    event::{ElementState, Event, KeyEvent, Modifiers, MouseScrollDelta, WindowEvent},
+    event_loop::{EventLoop, EventLoopWindowTarget},
+    keyboard::Key,
+    window::WindowBuilder,
+};
+
+use crate::{
+    clipboard,
+    cmd_line::SRGB_DEFAULT,
+    renderer::{build_context, build_window, GlWindow, WindowedContext},
+    window::{load_icon, SkiaRenderer, UserEvent},
+};
+
+const TEXT_COLOR: Color4f = WHITE;
+const BACKGROUND_COLOR: Color4f = BLACK;
+const FONT_SIZE: f32 = 12.0 * 96.0 / 72.0;
+const PADDING: f32 = 10.0;
+const MAX_LINES: i32 = 9999;
+const MIN_SIZE: PhysicalSize<u32> = PhysicalSize::new(800, 600);
+const DEFAULT_SIZE: PhysicalSize<u32> = PhysicalSize::new(500, 500);
+
+pub fn show_error_window(message: &str, event_loop: EventLoop<UserEvent>) {
+    let mut error_window = ErrorWindow::new(message, &event_loop);
+    error_window.run_event_loop(event_loop);
+}
+
+#[derive(Debug)]
+enum Scroll {
+    None,
+    Line(i32),
+    Page(f32),
+    Start,
+    End,
+}
+
+#[derive(EnumCount, EnumIter)]
+enum PossibleScrollDirection {
+    None,
+    Up,
+    Down,
+    Both,
+}
+
+struct Paragraphs {
+    message: Paragraph,
+    help_messages: [Paragraph; PossibleScrollDirection::COUNT],
+}
+
+struct ErrorWindow<'a> {
+    skia_renderer: SkiaRenderer,
+    context: WindowedContext,
+    font_collection: FontCollection,
+    size: PhysicalSize<u32>,
+    scale_factor: f64,
+    paragraphs: Paragraphs,
+    message: &'a str,
```

Comment:
It might be useful to add context about the fact that Neovide just crashed and this in fact represents an error which might be relevant to report, instead of just the error itself. Maybe something like this could be prepended?

> Neovide just crashed :(
> This is the error that caused the crash. In case you don't know what to do with this, please feel free to report this on https://github.com/neovide/neovide/issues!

Reply:
Done!

It's not perfect, since it also prints the message for user errors like giving the wrong `neovim-bin`. But regular wrong command line arguments are printed without that message.